AsyncDisplayKit/ASCellNode.mm

```diff
@@ -225,6 +225,10 @@ - (void)visibleStateDidChange:(BOOL)isVisible
 {
   [super visibleStateDidChange:isVisible];
   
+  if (isVisible && self.neverShowPlaceholders) {
+    [self recursivelyEnsureDisplaySynchronously:YES];
```

Comment:
@appleguy I moved this call. It used to live in the separate `willDisplayCell:` callback methods but I thought this a more appropriate place because in the previous case, the cell node wouldn't be in the hierarchy yet. So at the very least, this change gives us a tiny bit more time before we block the main thread. Thoughts?